### PR TITLE
feat(website): docs guide prover withdraw TTKO

### PR DESCRIPTION
**Pull request by MarcusWentz**

### CODE REVIEW

File: packages/website/pages/docs/guides/_meta.json [hunk 1]
```diff
@@ -23,6 +23,9 @@
   "enable-a-prover": {
     "title": "✅ Enable a prover"
   },
+  "prover-claim-ttko": {
+    "title": "🛄 Prover Claim TTKO"
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/_meta.json:27]: title's should follow sentence case convention. second thing is the phrasing could follow the convention (kind of like how the others start with a verb).. hmm.. maybe "Claim prover TTKO"?

File: packages/website/pages/docs/guides/prover-claim-ttko.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Prover Claim TTKO
+
+## Overview
+
+This guide will help you claim TTKO earned as a prover.
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/prover-claim-ttko.mdx:7]: would recommend a prerequisites section. something like "need to have successfully proven a block, see here how to enable a prover".

File: packages/website/pages/docs/guides/prover-claim-ttko.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Steps
+
+<Steps>
+
+### View TTKO prover address balance
+
+Go to the Taiko L1 contract read page and select [getTokenTokenBalance(addr address)](https://sepolia.etherscan.io/address/0x6375394335f34848b850114b66A49D6F47f2cdA8#readProxyContract#F10)
+to lookup your TTKO claim balance for your prover's address. 
+
+### Withdraw your TTKO
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/prover-claim-ttko.mdx:18]: maybe can expand this with two steps instead:
1. "Select withdrawTaikoToken and connect wallet
2. "Withdraw TTKO" -> this one should ultimately say "click the "Write"" button at the end

but basically just need that final click button step.

File: packages/website/pages/docs/guides/_meta.json (hunk 1 above)
Comment (MarcusWentz) [reply packages/website/pages/docs/guides/_meta.json]: Updated to "🛄 Claim prover TTKO".

File: packages/website/pages/docs/guides/prover-claim-ttko.mdx (hunk 2 above)
Comment (MarcusWentz) [reply packages/website/pages/docs/guides/prover-claim-ttko.mdx]: Update:
- You followed the [Enable a prover](/docs/guides/enable-a-prover) guide and have a prover which has proved a block to earn TTKO rewards.
- Must have a balance of ETH Sepolia (see: [Receive tokens](/docs/guides/receive-tokens)).

File: packages/website/pages/docs/guides/prover-claim-ttko.mdx (hunk 3 above)
Comment (MarcusWentz) [reply packages/website/pages/docs/guides/prover-claim-ttko.mdx]: Expanded into 2 steps now.

File: packages/website/pages/docs/guides/claim-prover-ttko.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Prover Claim TTKO
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/claim-prover-ttko.mdx:3]:
```suggestion
# Claim prover TTKO
```

Comment (MarcusWentz) [reply packages/website/pages/docs/guides/claim-prover-ttko.mdx]: Updated

File: packages/website/pages/docs/guides/claim-prover-ttko.mdx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Claim Prover TTKO
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/claim-prover-ttko.mdx:3]:
```suggestion
# Claim prover TTKO
```
nit: sentence case

**Review by dionysuzx — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**